grc/core/blocks/block.py

```diff
@@ -442,49 +440,35 @@ def format_expr(self, py_type):
         self.cpp_templates = copy.copy(self.orig_cpp_templates)
 
         # Determine the lvalue type
-        def get_type(element, _vtype):
+        def get_type(element: str, _vtype: typing.Optional[type] = None) -> str:
```

Comment:
Why `_` in `_vtype`?

Reply:
Left it as it was before me ([source](https://github.com/gnuradio/gnuradio/blob/d7f887227add25431b948d3a18471efd56bb1945/grc/core/blocks/block.py#L443)). Perhaps, it denoted an optional argument?